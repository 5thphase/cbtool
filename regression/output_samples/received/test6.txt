 status: Waiting for all active AIDRS daemons to finish gracefully....
 status: All AIDRS (daemons and objects were removed).
 status: Removing all VMCRS objects attached to this experiment.
 status: Removing all FIRS objects attached to this experiment.
 status: Removing all AI objects attached to this experiment.
 status: Removing all VM objects attached to this experiment.
 status: Removing all VMC objects attached to this experiment.
 status: VMC 6F6F6E22-C0A1-5ACE-BF56-B0854AFFA1CF was successfully unregistered on SimCloud "TESTSIMCLOUD".
 status: VMC 43C027B1-E2A5-5105-9687-A0DE0E22D961 was successfully unregistered on SimCloud "TESTSIMCLOUD".
 status: VMC 7E8B31AF-FFE6-587C-9802-69A072BAD1D5 was successfully unregistered on SimCloud "TESTSIMCLOUD".
 status: VMC 8B3B2305-49A5-50C3-BE6C-F50508162CBA was successfully unregistered on SimCloud "TESTSIMCLOUD".
 status: Done
 status: Removing all contents from Object Store (GLOBAL objects,VIEWS, etc.)
Disassociating default cloud: TESTSIMCLOUD
Cloud TESTSIMCLOUD was successfully detached from this experiment.
