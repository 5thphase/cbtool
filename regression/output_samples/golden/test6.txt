 status: Waiting for all active AIDRS daemons to finish gracefully....
 status: All AIDRS (daemons and objects were removed).
 status: Removing all VMCRS objects attached to this experiment.
 status: Removing all FIRS objects attached to this experiment.
 status: Removing all AI objects attached to this experiment.
 status: Removing all VM objects attached to this experiment.
 status: Removing all VMC objects attached to this experiment.
 status: VMC 64744820-92D0-5D65-B740-65F045C6B20A was successfully unregistered on SimCloud "TESTSIMCLOUD".
 status: VMC E6BAAB23-2BE0-5B4D-9719-0056F8198FBA was successfully unregistered on SimCloud "TESTSIMCLOUD".
 status: VMC 173B0882-0EF6-5D32-A977-891050096378 was successfully unregistered on SimCloud "TESTSIMCLOUD".
 status: VMC C58BB139-B926-5EBD-A684-1E9B469A1996 was successfully unregistered on SimCloud "TESTSIMCLOUD".
 status: Done
 status: Removing all contents from Object Store (GLOBAL objects,VIEWS, etc.)
Disassociating default cloud: TESTSIMCLOUD
Cloud TESTSIMCLOUD was successfully detached from this experiment.
